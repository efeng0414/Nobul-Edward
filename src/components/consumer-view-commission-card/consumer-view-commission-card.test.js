import React from "react";
import renderer from "react-test-renderer";
import { IntlProvider } from "react-intl";

import ConsumerViewCommissionCardWrapped from "./index";

describe("ConsumerViewCommissionCard", () => {
  const {
    WrappedComponent: ConsumerViewCommissionCard
  } = ConsumerViewCommissionCardWrapped;

  const componentProps = {
    intl: global.getIntl()
  };

  const componentJSX = <ConsumerViewCommissionCard {...componentProps} />;

  describe("Component Snapshot", () => {
    const snapshotJSX = (
      <IntlProvider {...global.intl}>{componentJSX}</IntlProvider>
    );

    const componentSnapshot = renderer.create(snapshotJSX).toJSON();
    it("should matched stored snapshot", () => {
      expect(componentSnapshot).toMatchSnapshot();
    });
  });
});
