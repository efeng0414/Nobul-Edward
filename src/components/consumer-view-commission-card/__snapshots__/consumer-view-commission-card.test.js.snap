// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConsumerViewCommissionCard Component Snapshot should matched stored snapshot 1`] = `
<div
  className="commission-card"
>
  I AM A COMMISION CARD
</div>
`;
